packages/microcode/microcode.spec

```
@@ -17,27 +17,40 @@ License: LicenseRef-scancode-amd-linux-firmware-export AND LicenseRef-scancode-i
 # RPM only allows one URL field per package, so this is about as accurate as we
 # can be. The real upstream URLs for AMD and Intel microcode are given below in
 # the subpackage definitions.
-URL: https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/microcode
+URL: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/tree/develop/packages/microcode
 
 Source0: https://www.kernel.org/pub/linux/kernel/firmware/linux-firmware-%{amd_ucode_version}.tar.xz
 Source1: https://www.kernel.org/pub/linux/kernel/firmware/linux-firmware-%{amd_ucode_version}.tar.sign
 Source2: gpgkey-4CDE8575E547BF835FE15807A31B6BD72486CFD6.asc
 Source3: https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/archive/refs/tags/microcode-%{intel_ucode_version}.tar.gz
 
 # Lets us install "microcode" to pull in the AMD and Intel updates.
-Requires: %{_cross_os}microcode-amd
-Requires: %{_cross_os}microcode-intel
+Requires: %{_cross_os}microcode-amd-ec2
+Requires: %{_cross_os}microcode-intel-other
+Requires: %{_cross_os}microcode-intel-ec2
 
 %description
 %{summary}.
 
-%package amd
-Summary: Microcode for AMD processors
+# Microcode applicable to Intel and AMD processors in EC2
+%package ec2
+Summary: Microcode for EC2 AMD and Intel processors
```

Rephrase so it doesn't sound like these are "EC2[-specific] AMD processors"
```suggestion
Summary: Microcode for AMD and Intel processors available in EC2
```